sources/can-clab.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Canada's National Park Boundaries dataset", 
+    "edtf:inception": "2021-08-31", 
+    "fullname": "National Framework Canada Lands Administrative Boundaries", 
+    "id": 1746270541, 
+    "key": "", 
+    "license": "https://open.canada.ca/en/open-government-licence-canada", 
+    "license_text": "You are free to: copy, modify, publish, translate, adapt, distribute or otherwise use the Information in any medium, mode or format for any lawful purpose.", 
```

Comment:
Their website says the credit text should say: 

> `Contains information licensed under the Open Government Licence – Canada.` 

Should that go in the license text field or somewhere else?

Reply:
Ah, good catch. I've added this blurb to the `license_text` in this source json.